includes/api/order.php

```diff
@@ -95,6 +95,12 @@ function createWcOrder(WP_REST_Request $request)
         if (is_wp_error($orderId)) {
             $checkout_error = $orderId->get_error_message();
         }
+        //Keep order in draft status untill customer info available
+        wp_update_post(array(
```

Comment:
Can we add this status in the initial create order? Any new db update may add lot of delay for slow merchants

Reply:
During the order creation by default, wooc will set the status to wc-pending. No option is available to create it in draft status.